oss_src/unity/python/sframe/data_structures/sframe.py

```diff
@@ -1641,6 +1631,36 @@ def read_json(cls,
                 return g
         else:
             raise ValueError("Invalid value for orient parameter (" + str(orient) + ")")
+    
+    @classmethod
+    def set_spark_integration_jar_path(cls, jar_path):
+        """
+        Set the absolute path of the jar file required to enable GraphLab Create's
+        integration with Apache Spark.
+        """
+        global SFRAME_GRAPHLABUTIL_REF
+        if not isinstance(jar_path, str):
+            raise TypeError("The jar path must be str type")
+        if not os.path.isfile(jar_path):
+            raise RuntimeError("Cannot find the file %s" % jar_path)
+        if not jar_path.endswith(".jar"):
+            raise RuntimeError("The jar path must point to a jar file")
+        if SFRAME_GRAPHLABUTIL_REF is not None:
+            raise RuntimeError("The jar path should be set before calling to_rdd(), from_rdd(), or to_spark_dataframe() methods.")
+
+        BINARY_PATHS['RDD_JAR_PATH'] = jar_path
+
+    @classmethod
+    def get_spark_integration_jar_path(cls):
+        """
+        The absolute path of the jar file required to enable GraphLab Create's
+        integration with Apache Spark.
+        """
+        if 'RDD_JAR_PATH' not in BINARY_PATHS:
+            raise RuntimeError("Could not find a spark integration jar.  "\
+                "Does your version of GraphLab Create support Spark Integration (is it >= 1.0)?")
```

Comment:
Is it that we cant find it, or is it that things are just not set yet?   Is there any possible way that the resolution of this is just to call set_spark_integration_jar_path?  If so, then this error message should reflect that.